src/material/button/button.md

```diff
@@ -33,10 +33,22 @@ screen-readers, which will read the text character-by-character. We leave the de
 approach this to the consuming app.
 
 ### Accessibility
-Angular Material uses native `<button>` and `<a>` elements to ensure an accessible experience by
-default. The `<button>` element should be used for any interaction that _performs an action on the
-current page_. The `<a>` element should be used for any interaction that _navigates to another
-view_.
+Angular Material uses native <button> and <a> elements. Both of the two look the same when applied Material Design Styling, but have different semantic meaning for assitive technology. To ensure an accessible experience, choose the element which is most approprite for the purpose of the interaction.
 
-Buttons or links containing only icons (such as `mat-fab`, `mat-mini-fab`, and `mat-icon-button`)
-should be given a meaningful label via `aria-label` or `aria-labelledby`.
+#### Performing an Action
+Use the `<button>` element for interactions that _perform an action on the current page_.
+ - submitting a form
+ - sending a chat message
+ - copying content to the clipboard
+
+#### Navigation
+Use the `<a>` element for interactions that _navigate to another view_.
+ - navigating to a different page
```

Comment:
I think that these two sections could convey their points more concisely and in paragraph form without the bulleted lists. Something like

```markdown
#### Choose the right element
You should use a button element for any action
that does not navigate to another URL, such as opening
a dialog or saving data. You should use an anchor element
for any interaction that navigates to another URL using either
`href` or Angular's `routerLink` directive.
```